src/telliot_feeds/reporters/reporter_autopay_utils.py

```diff
@@ -264,11 +289,20 @@ async def reward_claim_status(self, require_success: bool = False) -> Any:
                             )
 
         multi_call = Multicall(calls=reward_claimed_status_call, _w3=self.w3, require_success=require_success)
```

Comment:
All these `mutli_call` reused var names should be made specific to each one, so the code's easier to read/understand. Like `multicall_claim_rewards` or something

Reply:
I agree thats my bad, I'll keep this in mind.  Also just a heads up I'll be making a lot of changes on this file for 360 updates and I'll be more conscious about the var names.